ClLike/cl_like/power_spectrum.py

```diff
@@ -53,6 +53,9 @@ class Pk(Theory):
     nonlinear_emu_path = None
     nonlinear_emu_details = None
     use_baryon_boost : bool = False
+    ignore_lbias : bool = False 
+    allow_bcm_emu_extrapolation_for_shear : bool = True
+    allow_halofit_extrapolation_for_shear : bool = False
```

Comment:
Comment for future. It this keeps growing it could be good to have an attribute "ptc_extra_args" or sth like that to pass the arguments defined for each perturbation theory calculator, since I guess the will not be, in general, usable in the others ones.